### PR TITLE
Add virtual comp and tech day reminder email

### PR DESCRIPTION
**Pull request by RealOrangeOne**

### CODE REVIEW

File: SR2023/2023-02-23-virtual-competition-reminder.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+to: Student Robotics 2023
+subject: SR2023 Virtual Competition and Tech Day Reminder
+---
+
+Hi
+
+We're just reminding you about the [virtual competition](https://studentrobotics.org/events/sr2023/virtual-competition/) happening this weekend.
```
Comment (PeterJCLaw) [inline SR2023/2023-02-23-virtual-competition-reminder.md:8]: Suggestion for a slightly more upbeat intro:
```suggestion
We're really looking forward to see your robots in the [virtual competition](https://studentrobotics.org/events/sr2023/virtual-competition/) this weekend.
```

File: SR2023/2023-02-23-virtual-competition-reminder.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+[Submit your robot code](https://studentrobotics.org/code-submitter/).
```
Comment (PeterJCLaw) [inline SR2023/2023-02-23-virtual-competition-reminder.md:10]:
```suggestion
[Submit your robot code](https://studentrobotics.org/code-submitter/) by 8pm on Friday.
```

File: SR2023/2023-02-23-virtual-competition-reminder.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+If you're having any issues logging in to the code submitter, please let us know as soon as possible. Details of how to upload can be found in our previous email and Discord.
```
Comment (PeterJCLaw) [inline SR2023/2023-02-23-virtual-competition-reminder.md:12]: Can we repeat the instructions here? They're pretty simple as I recall?
```suggestion
If you're having any issues logging in to the code submitter please let us know as soon as possible. You can log in to the code submitter using your TLA and the same password as you use for Discord.
```

**Review by PeterJCLaw — COMMENTED** [review]
Generally good, some small suggestions.

File: SR2023/2023-02-23-virtual-competition-reminder.md (hunk 2 above)
Comment (prophile) [reply SR2023/2023-02-23-virtual-competition-reminder.md:10]: UK time?

File: SR2023/2023-02-23-virtual-competition-reminder.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+You have until 8pm (UK time) Friday 24th February to submit your team's latest code. You can upload early, but at that time we'll capture all the uploads. If you fail to upload any code, you will be marked as absent from all the matches and score 0 points. This will not affect your ability to compete in the [final competition](https://studentrobotics.org/events/sr2023/competition/).
+
+You won't be able to re-submit between matches in the virtual competition, so it's important your code is as well-tested and reliable as possible come the submission deadline. Because of this, we have a few suggestions for things to keep in mind:
+
+- Remember to use [`R.zone`](https://studentrobotics.org/docs/programming/sr/#OtherRobotAttributes) - as your robot could start in any of the four corners, not just corner 0.
+- Make sure you're using [available libraries](https://studentrobotics.org/docs/programming/python/libraries#simulator).
```
Comment (prophile) [inline SR2023/2023-02-23-virtual-competition-reminder.md:19]: +only?

**Review by prophile — DISMISSED** [review]

File: SR2023/2023-02-23-virtual-competition-reminder.md (hunk 2 above)
Comment (PeterJCLaw) [reply SR2023/2023-02-23-virtual-competition-reminder.md:10]: The full detail (date & time) is included in the following paragraph, so I wasn't sure we needed it here as well. I don't feel strongly though.

Comment (RealOrangeOne) [reply SR2023/2023-02-23-virtual-competition-reminder.md:10]: As it's in the next paragraph, I think it's ok to leave out here.

Comment (PeterJCLaw) [reply SR2023/2023-02-23-virtual-competition-reminder.md:10]: The timezone or the time entirely? My thought was to make it explicit what the deadline was.

Comment (RealOrangeOne) [reply SR2023/2023-02-23-virtual-competition-reminder.md:10]: We mention the time and the timezone 2 paragraphs later, I don't think we need it in this main CTA. Alternatively, we inline the link into that paragraph and reflow the intermediary paragraph into it

File: SR2023/2023-02-23-virtual-competition-reminder.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+Hi
+
+We're really looking forward to see your robots in the [virtual competition](https://studentrobotics.org/events/sr2023/virtual-competition/) this weekend.
+
+[Submit your robot code](https://studentrobotics.org/code-submitter/).
+
+If you're having any issues logging in to the code submitter please let us know as soon as possible. You can log in to the code submitter using your TLA and the same password as you use for Discord.
+
+You have until 8pm (UK time) Friday 24th February to submit your team's latest code. You can upload early, but at that time we'll capture all the uploads. This is your first 4 appearances in the competition, scores you real points, and is a great way to see how you robot's strategy pays off. This will not affect your ability to compete in the [final competition](https://studentrobotics.org/events/sr2023/competition/).
```
Comment (sedders123) [inline SR2023/2023-02-23-virtual-competition-reminder.md:14]:
```suggestion
You have until 8pm (UK time) Friday 24th February to submit your team's latest code. You can upload early, but at that time we'll capture all the uploads. These matches are your first 4 appearances in the competition, scoring you valuable league points. It's also a great way to see how you robot's strategy fares against other robots. Not entering will not affect your ability to compete in the [final competition](https://studentrobotics.org/events/sr2023/competition/) but will put you at a disadvantage to the teams that have entered our virtual event.
```

Comment (PeterJCLaw) [inline SR2023/2023-02-23-virtual-competition-reminder.md:14]: Maybe even drop the bit about the final competition?

Comment (sedders123) [reply SR2023/2023-02-23-virtual-competition-reminder.md:14]: Comment from Peter, we could drop the last sentence entirely. I think I agree but up to you @RealOrangeOne

File: SR2023/2023-02-23-virtual-competition-reminder.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
+---
+to: Student Robotics 2023
+subject: SR2023 Virtual Competition and Tech Day Reminder
+---
+
+Hi
+
+We're really looking forward to see your robots in the [virtual competition](https://studentrobotics.org/events/sr2023/virtual-competition/) this weekend.
```
Comment (sedders123) [inline SR2023/2023-02-23-virtual-competition-reminder.md:8]:
```suggestion
We're really looking forward to seeing your robots in the [virtual competition](https://studentrobotics.org/events/sr2023/virtual-competition/) this weekend.
```

**Review by sedders123 — APPROVED** [review]

**Pull request merged**